#!/usr/bin/python3
def raise_exception():
    a = "c"
    a = a + 1
    print ("{:d}".format(a))
